### PR TITLE
comments

### PR DESCRIPTION
**Pull request by bjoveski**

### CODE REVIEW

File: java/difference-of-squares/src/main/java/DifferenceOfSquaresCalculator.java [hunk 1]
```diff
@@ -4,23 +4,23 @@ public DifferenceOfSquaresCalculator(){}
 
 	public int computeSquareOfSumTo(int number){
 		int solution = 0;
-		for(int i=1; i<=number;i++) solution+=i;
```
Comment (bjoveski) [inline java/difference-of-squares/src/main/java/DifferenceOfSquaresCalculator.java:7]: spacing issues - always be consistent (no spaces around the `=`, `<=` in fors, spaces after the ;

Comment (bjoveski) [inline java/difference-of-squares/src/main/java/DifferenceOfSquaresCalculator.java:7]: I always like to surround the for/if, etc. in {}. even for single line stuff. it is more readable and avoid silly mistakes when you need to change the logic to something more than a single line

File: java/difference-of-squares/src/main/java/DifferenceOfSquaresCalculator.java [hunk 2]
```diff
@@ -4,23 +4,23 @@ public DifferenceOfSquaresCalculator(){}
...
-		return solution*solution;
+		for(int i=1; i<=number; i++){
+			solution += i;
```
Comment (bjoveski) [inline java/difference-of-squares/src/main/java/DifferenceOfSquaresCalculator.java:8]: spacing around += here

File: java/difference-of-squares/src/main/java/DifferenceOfSquaresCalculator.java [hunk 3]
```diff
@@ -4,23 +4,23 @@ public DifferenceOfSquaresCalculator(){}
...
+		for(int i=1; i<=number; i++){
+			solution += i*i;
+		}
 		return solution;
 	}
 
-
 	public int computeDifferenceOfSquares(int number){
-
-		return computeSquareOfSumTo(number)-computeSumOfSquaresTo(number);
```
Comment (bjoveski) [inline java/difference-of-squares/src/main/java/DifferenceOfSquaresCalculator.java:21]: this line is too complicated. avoid calling functions and returning the result in the same line. what we have below is more readable IMO

**Review by Matzy93 — APPROVED** [review]
Thanks!

**Comment by bjoveski** [comment]
heheh :)

**Pull request merged**